Meta/include/meta.h

```diff
@@ -25,6 +25,18 @@
 #include <DRoot.h>
 #include <ASNodeManager.h>
 #include <DStandardMetaData.h>
+#include <ASNodeQueries.h>
 
 Device::DStandardMetaData* configureMeta(Configuration::Configuration & config, AddressSpace::ASNodeManager *nm, UaNodeId parentNodeId);
 void destroyMeta (AddressSpace::ASNodeManager *nm);
+
+template<typename TAddressSpaceType>
```

Comment:
Pure cosmetics, If we could be consistent with already existing convention of template class names in quasar which doesnt use "T" as prefix, that would be better.

Reply:
done